### PR TITLE
kubernetes: prep for beta-kubernetes

### PR DESCRIPTION
**Pull request by dkoshkin**
## Description

The upcoming `kubernetes` version will be released under `beta-kubernetes` this PR is in preparation for that.

- [x] Remove `beta-kubernetes` update-release scripts and extend the `kubernetes` one to support variable package names
- [x] Remove old `beta-kubernetes` docs 
- [x] Fix `kubernetes` update-release scripts to properly work when skipping versions

## Urgency
- [ ] Blocker 
- [x] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11, 1.12).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

**Comment by dkoshkin** [comment]
To update `beta-kubernetes` docs you can run something like(will also work with tags with the other script):
```
./scripts/service-update-scripts/kubernetes-merge-branch.sh beta-kubernetes-docs "" beta-kubernetes
```
First param is the branch name in `dcos-kubernetes` 
Second param is the versions to skip - none in this case
Third param is the package name

File: scripts/service-update-scripts/kubernetes-merge-branch.sh [hunk 1]
```diff
@@ -30,7 +32,7 @@ git checkout dcos-kubernetes/$branch docs/package
 # remove any user specified directories 
 if [ -n "$skip" ]; then 
   echo "Skipping $skip"
-  for d in $(echo $skip | sedi "s/,/ /g"); do rm -rf docs/package/$d; done
+  for d in $(echo $skip | sed "s/,/ /g"); do rm -rf docs/package/$d; done
```
Comment (bmcstdio) [inline scripts/service-update-scripts/kubernetes-merge-branch.sh]: Will this work as expected in macOS?

File: scripts/service-update-scripts/kubernetes-merge.sh [hunk 2]
```diff
@@ -30,7 +32,7 @@ git checkout tags/$version docs/package
 # remove any user specified directories 
 if [ -n "$skip" ]; then 
   echo "Skipping $skip"
-  for d in $(echo $skip | sedi "s/,/ /g"); do rm -rf docs/package/$d; done
+  for d in $(echo $skip | sed "s/,/ /g"); do rm -rf docs/package/$d; done
```
Comment (bmcstdio) [inline scripts/service-update-scripts/kubernetes-merge.sh]: _Idem_.

File: scripts/service-update-scripts/kubernetes-merge-branch.sh (hunk 1 above)
Comment (dkoshkin) [reply scripts/service-update-scripts/kubernetes-merge-branch.sh]: Yes, `sedi`( `-i` ) is only needed when working with files, this is using `stdout` so `-i` actually breaks it on MacOS.

**Pull request merged**